packages/client/src/index.ts

```diff
@@ -89,8 +89,7 @@ export function decorate (
 ): DecoratedTransfer {
   const type = getTransferType(transfer)
 
-  // not needed with structNullChecks false: @ts-expect-error
-  // to be improved
+  // @ts-expect-error
```

Comment:
why is this required here?

Reply:
Because `locale` is optional. There is another PR in draft (https://github.com/aurora-is-near/rainbow-bridge-client/pull/19) so I didn't want to modify this piece of logic in this PR and re-added `@ts-expect-error` since using strictNullChecks in tsconfig will cause `Type 'undefined' cannot be used as an index type.ts(2538)`.